spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/DefaultRestTemplateConfig.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2013-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.netflix.eureka;
+
+import java.util.Objects;
+
+public class DefaultRestTemplateConfig implements RestTemplateConfig {
+
+	private int connectTimeout = 10 * 1000;
+
+	private int connectRequestTimeout = 10 * 1000;
+
+	private int socketTimeout = 10 * 1000;
+
+	@Override
+	public int getConnectTimeout() {
+		return connectTimeout;
+	}
+
+	@Override
+	public int getConnectRequestTimeout() {
+		return connectRequestTimeout;
+	}
+
+	@Override
+	public int getSocketTimeout() {
+		return socketTimeout;
+	}
+
+	void setConnectTimeout(int connectTimeout) {
```

Comment:
Since this is a property bean, I'd make the setters public.

Reply:
Oh, I should have checked the access modifier... Thanks 👍🏻